app/app/urls.py

```diff
@@ -178,6 +178,7 @@
     # brochureware views
     url(r'^about/?', retail.views.about, name='about'),
     url(r'^mission/?', retail.views.mission, name='mission'),
+    url(r'^ui/?', retail.views.ui, name='ui'),
```

Comment:
Please use `re_path` since we are trying to migrate completely from `url`

Reply:
^